README.md

```diff
@@ -17,5 +17,5 @@ All values passed to `--arg` are the JSON representation of SCVals.
 ## Example
 
 ```
-soroban invoke --id <HEX_CONTRACTID> --wasm <WASMFILE> --fn <FUNCNAME> --arg 32 --arg 4
+soroban-cli invoke --id <HEX_CONTRACTID> --wasm <WASMFILE> --fn <FUNCNAME> --arg 32 --arg 4
```

Comment:
❓ Why change the soroban binary from `soroban`, back to `soroban-cli`? We recently made the change to remove the `-cli` which came from both internal feedback and external community feedback.